.github/workflows/dockerbuild.yaml

```diff
@@ -122,3 +122,39 @@ jobs:
         repository: ibm-cloud-architecture/refarch-kc-gitops
         event-type: gitops-refresh
         client-payload: '{"ref": "${{ github.ref }}", "sha": "${{ github.sha }}", "source": "${{ github.repository }}"}'
+
+  update-gitops:
+    needs:
+      build-docker-images
+    runs-on: ubuntu-latest
+    steps:
+    - name: Configure git client
+      id: configure-git
+      uses: djones6/git-config-action@master
+      with:
+        user-email: ${{ secrets.GITOPS_EMAIL }}
+        gitops-token: ${{ secrets.GITOPS_TOKEN }}
+    - name: Retrieve fleet-ms app-deploy.yaml
+      uses: actions/download-artifact@v2
+      with:
+        name: fleetms-app-deploy.yaml
+        path: fleetms
+    - name: Update fleet-ms app-deploy.yaml in GitOps repo
+      id: update-fleet-gitops
+      uses: djones6/appsody-gitops-update-action@master
+      with:
+        service-name: fleetms
+        github-org: ibm-cloud-architecture
```

Comment:
Note, it would be nicer if we could use a built-in variable here, the closest I could find however was `GITHUB_REPOSITORY` which would resolve to `ibm-cloud-architecture/refarch-kc-ms` and I wasn't sure it was worth trying to extract it from here.